### PR TITLE
Doc: Add project information and setup instructions to README file

### LINKED ISSUE #42: Create README documentation
## Description
As a contributor,
I need know how to run the project,
so that I can contribute to it and run it locally.

Currently, README is empty, so a newcomer that wishes to contribute to the project may not know how to run the project.

### Update [Required]
- [ ] Update README to contain the setup and run instructions of the app, everything one needs to run this project
- [ ] Have a description of the application and its tech stack

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
3 hours

### PR DESCRIPTION
**Pull request by mtreacy002**
### Description
Add project information and setup instructions for running BridgeInTech web application to README file

Fixes #42 

### Type of Change:
- Documentation

**Code/Quality Assurance Only**
NA

### How Has This Been Tested?
View the proposed page on fork repo and confirm it rendered as expected.

### Checklist:
- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by mtreacy002** [comment]
Update @isabelcosta and @anitab-org/bridgeintech-maintainers. Here's the updated README file with added project information and setup instructions. Please check and provide feedback. Thanks

**Review by ramitsawhney27 — DISMISSED** [review]

**Review by isabelcosta — CHANGES_REQUESTED** [review]
@mtreacy002 you are missing the run instructions. You are saying here how to setup the backend but I still don't see how to run the web project and where to see it live. Is it `npm install` and then `npm start`?

**Comment by mtreacy002** [comment]
> @mtreacy002 you are missing the run instructions. You are saying here how to setup the backend but I still don't see how to run the web project and where to see it live. Is it `npm install` and then `npm start`?

Sorry, will fix it now

**Comment by mtreacy002** [comment]
Update @isabelcosta . I've added another section to README file: `Run the app`. Please re-review. Thanks

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+<img width="1348" alt="Screen Shot 2020-07-19 at 12 05 25 pm" src="https://user-images.githubusercontent.com/29667122/87865402-3e9f1580-c9b8-11ea-97e1-6dc24fdc970f.png">
+
+**Important**
+Please be aware that there are **drawbacks** if you chose to run the BIT and MS backend servers remotely (**option 2**):
+* There will be a **significant delay** from the time when the request is made on the BIT web GUI to the time the response is received. This is because BIT Heroku makes calls to MS-for-BIT Heroku backend server for MS related functionalities. Both of these Heroku servers are running on [free `dynos`](https://www.heroku.com/dynos) that have limited capacity. When you run both BIT and MS-for-BIT servers locally, you will not have this time lag issue.
+* The BIT Heroku baackend server will only carries the latest code that have been approved and merged to the develop branch. This means, if you are working on a PR or testing an open PR, this **Option 2** (running remote BIT Heroku server) **MUST NOT BE USED**.     
+
+## Run the app
+
+Now that you have set the MS backend, BIT backend and frontend servers, you can run BIT React application by running `npm start`.
```
Comment (isabelcosta) [inline README.md]: Can a contributor run `npm start` before `npm install`?
Isn't `npm install` for installing dependencies on web proj? @mtreacy002

Comment (mtreacy002) [reply README.md]: @isabelcosta , I don't remember if I have ever tried running the frontend before installing the dependencies. But from running frontend through this project, I notice if there's any dependency missing, the app might not render well (or might not render at all). Perhaps similar to Swagger UI often won't render if there's any dependencies missing (even before that, python console will crash if we don't install/update dependencies on the requirements.txt). That's why I placed the npm start after all the backend/frontend setup done.

Comment (isabelcosta) [reply README.md]: All I am saying is that the `npm install` step is missing. And if could put a 3rd step with the localhost link, people have to go to, that would link to the page. Also a numbered list is easier to read :)

Comment (mtreacy002) [reply README.md]: Done. Thanks for pointing that out, @isabelcosta . I've made the changes and have pushed the code again. Could you please approve it now? Thanks

**Review by isabelcosta — APPROVED** [review]
Great work @mtreacy002 ! Now newcomers can learn how to work with BIT code 🎉

**Review by foongminwong — APPROVED** [review]
👍👍👍

**Pull request merged**